sentry/src/main/java/io/sentry/MonitorConfig.java

```diff
@@ -15,13 +15,23 @@ public final class MonitorConfig implements JsonUnknown, JsonSerializable {
   private @Nullable Long checkinMargin;
   private @Nullable Long maxRuntime;
   private @Nullable String timezone;
-
+  private @Nullable Long failureIssueThreshold;
+  private @Nullable Long recoveryThreshold;
   private @Nullable Map<String, Object> unknown;
 
   public MonitorConfig(final @NotNull MonitorSchedule schedule) {
     this.schedule = schedule;
   }
 
+  public MonitorConfig(
```

Comment:
`m` We don't have a ctor overload for other fields except `schedule`. Do you think we should add an overload for the new ones?

Reply:
Good question, I don't think we really need them.
I'll remove the additional ctor, we can re-add some additional ctors if the need arises.